src/plugins/intel_cpu/src/transformation_pipeline.h

```diff
@@ -41,6 +43,7 @@ class Transformations {
     const bool    enableLpt;
     const bool    enableBF16;
     const bool    isLegacyApi;
+    const Config& config;
```

Comment:
compiler reported this field is not used
looks like we need to make it happy differently.

have you tried to compile with clang?

Reply:
by the way, there are still warnings in the build, after -Wall was enabled.
Are there any plans to activation -Werror or something?

Reply:
@ilya-lavrenov thanks for the suggestion. I compiled the project without debug caps and fixed "unused field" warning.

Reply:
> by the way, there are still warnings in the build, after -Wall was enabled.
Are there any plans to activate -Werror or something?

We already use CMAKE_COMPILE_WARNING_AS_ERROR on CI